packages/wdio-browserstack-service/src/util.ts

```diff
@@ -419,6 +427,23 @@ export const launchTestSession = o11yErrorHandler(async function launchTestSessi
     }
 })
 
+export const validateCapsWithAppA11y = (platformMeta?: { [key: string]: any; }) => {
+    /* Check if the current driver platform is eligible for AppAccessibility scan */
+    BStackLogger.debug(`platformMeta ${JSON.stringify(platformMeta)}`)
+    try {
+        if (platformMeta?.platform_name && String(platformMeta?.platform_name).toLowerCase() === 'android') {
+            if (platformMeta?.platform_version && parseInt(platformMeta?.platform_version?.toString()) < 11) {
+                BStackLogger.warn('App Accessibility Automation tests are supported on OS version 11 and above for Android devices.')
+                return false
+            }
+        }
+        return true
```

Comment:
Are you sure `parseInt(platformMeta?.platform_version?.toString())` will never return `NaN`? You maybe want to do some more sophisticated version checking here.

```suggestion
        if (
            (platformMeta?.platform_name && String(platformMeta?.platform_name).toLowerCase() === 'android') &&
            (platformMeta?.platform_version && parseInt(platformMeta?.platform_version?.toString()) < 11)
        ) {
                BStackLogger.warn('App Accessibility Automation tests are supported on OS version 11 and above for Android devices.')
                return false
            }
        }
        return true
```